src/sample/loginController.java

```diff
@@ -71,7 +71,15 @@ public void SignIn(MouseEvent event) throws IOException {
             courseRegController.inputID(id.getText());
             downloadController.inputID(id.getText());
             switchTo(event, "stuDash.fxml");
+
+            stuDashController.getID(id.getText());
+          } else if (rbLecturer.isSelected()) {
+            switchTo(event, "lecDash.fxml");
+          } else {
+            loginMessage.setText("Please choose Student or Lecturer!");
+
```

Comment:
your loginController.java is outdated, should update with the current one

Reply:
i already fetch and merge the latest version, i am now working in the latest version

Reply:
this is not the latest version

Reply:
ya i mean i am now doing on the latest version for lec dashboard, later I will push the code

Reply:
change it to the latest one